assets/eip-6617/contracts/EIP6617Meta.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.7;
+
+import "./interfaces/IEIP6617Meta.sol";
+
+/**
+ * @dev Implement the metadata of EIP-6617
+ */
+contract EIP6617Meta is IEIP6617Meta {
```

Comment:
This implementation doesn't implement what is being specified.

Reply:
Hi @xinbenlv, we will review then update the EIP to match the requirements. Could you be more specific so we can update all at once?.